week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -14,38 +14,23 @@ and have different levels of positivity or negativity.
 
 Below are the possible answers:
 
-## Very positive
-It is certain.
-It is decidedly so.
-Without a doubt.
-Yes - definitely.
-You may rely on it.
-
-## Positive
-As I see it, yes.
-Most likely.
-Outlook good.
-Yes.
-Signs point to yes.
-
-## Negative
-Reply hazy, try again.
-Ask again later.
-Better not tell you now.
-Cannot predict now.
-Concentrate and ask again.
-
-## Very negative
-Don't count on it.
-My reply is no.
-My sources say no.
-Outlook not so good.
-Very doubtful.
+// This should log "The ball has shaken!"
 */
 
-// This should log "The ball has shaken!"
-// and return the answer.
-function shakeBall() {}
+function shakeBall() {
```

Comment:
You have all of the pieces you need here, but they're not _quite_ in the right places :) If you check the comments again, `shakeBall` currently returns the category of answer (e.g ."very positive") but _should_ return the actual answer (e.g. "It is certain."), and `checkAnswer` currently returns the actual answer (e.g. "It is certain.") but _should_ return what the category of answer was (e.g. "very positive"). Can you try swapping things around so that they work that way around?